packages/devtools_app/lib/src/charts/flame_chart.dart

```diff
@@ -379,11 +379,7 @@ abstract class FlameChartState<T extends FlameChart,
   }
 
   void _handleKeyEvent(RawKeyEvent event) {
-    if (event.isAltPressed) {
-      _altKeyPressed = true;
-    } else {
-      _altKeyPressed = false;
-    }
+    _altKeyPressed = event.isAltPressed ? true : false;
```

Comment:
This should be further simplified to `_altKeyPressed = event.isAltPressed;` (as the branches just return the value of the condition).

Reply:
done. filed https://github.com/dart-lang/linter/issues/3888 to track this as this really obvious unnecessary code has been committed over 200X in g3.